src/components/EmailNotifications/DeleteEmail.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from 'react'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+
+import emailNotifcationIllustration from '../../assets/emailNotifications.svg'
+
+const DeleteEmail = React.memo(function DeleteEmail({
+  isModal,
+  email,
+  onDelete,
+  onCancel,
+}) {
+  const theme = useTheme()
+
+  const handleOnDelete = useCallback(() => {
+    onDelete()
+  }, [onDelete])
+
+  const handleOnCancel = useCallback(async () => {
+    onCancel()
+  }, [onCancel])
```

Comment:
We don't need these—the responsibility of memoizing them should be at the parent.